app/views/state_file/state_file_pages/about_page.html.erb

```diff
@@ -1,18 +1,18 @@
 <% title = t(".header") %>
-<h1 class="h2 strong" style="font-weight: bold"><%= title %></h1>
-<div>
-  <%= t(".subheader_html") %>
-</div><br>
+
+<div class="fyst-home-container">
+  <div class="fyst-home-content">
+    <h1 class="h1 strong" style="font-weight: bold"><%= title %></h1>
+  </div>
+  <%= image_tag 'questions/welcome.svg', class: 'fyst-home-image' %>
+</div>
+
 <div>
-  <%= link_to t(".action1_text"), "" %>
+  <%= t(".subheader_html", login_path: login_options_path(us_state: "az")) %>
 </div><br>
-<div>
-  <%= t(".action2") %>
-  <%= link_to t(".action2_text"), "" %>
-</div>
 
 <div class="blue-group spacing-below-15 spacing-above-15">
-  <%= t(".section1_html", direct_file_url: '') %>
+  <%= t(".section1_html", faq_link: state_faq_path(us_state: 'az')) %>
```

Comment:
Ditto here or we make an intermediate page to request the state.

Reply:
There is no "us" faq page - but instead we use `root_path(anchor: "common_questions")`

Reply:
Done